gitops/kcp/gateway-bindings/compute-apibinding.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: apis.kcp.dev/v1alpha1
+kind: APIBinding
+metadata:
+  name: cluster-workspace
+spec:
+  reference:
+    workspace:
+      exportName: kubernetes
```

Comment:
Shouldn't we give a more descriptive name by default, such as `pipeline-service`?

Reply:
The APIBinding is for our compute apiexport. What do you think of pipeline-service-compute? This would be to avoid confusion with APIBindings created for controllers.

Reply:
Works for me :wink: